qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/FindPatientPage.java

```diff
@@ -27,6 +28,7 @@ public void enterPatient(String patient) {
 	}
 
 	public ClinicianFacingPatientDashboardPage clickOnFirstPatient() {
+		waiter.until(ExpectedConditions.elementToBeClickable(PATIENT_NAME_SEARCH_RESULT));
```

Comment:
Is there any reason why adding a waiter call here?

Reply:
The reason is because it fails with out it. the element takes morethan 15ms to be clickable. thats why i have to  add it there.

Reply:
Is this failing in actions and ci?

Reply:
Is there any exiting test that uses the method(`clickObFirstPatient()`) that is failing in github actions?